twake/backend/core/src/Twake/Users/Services/User.php

```diff
@@ -335,7 +336,8 @@ public function generateJWT($user, $workspaces = []){
             "type" => "refresh",
             "iat" => intval(date("U")) - 60*10,
             "nbf" => intval(date("U")) - 60*10,
-            "sub" => $user->getId()
+            "sub" => $user->getId(),
+            "csl_sub" => $user->getIdentityProviderId(),
```

Comment:
How bad is this name? `csl` means console, right? Why not just naming it `provider`?

Reply:
`provider_id` then ?